spotbugs-ant/src/main/java/edu/umd/cs/findbugs/anttask/AbstractFindBugsTask.java

```diff
@@ -388,8 +390,23 @@ protected void addArg(String arg) {
      * Sets the given string to be piped to standard input of the FindBugs JVM
      * upon launching.
      */
+    protected void appendToInputString(String input) {
+        if (null == inputStringBuffer) {
+            inputStringBuffer = new StringBuffer();
+        }
+        inputStringBuffer.append(input);
+    }
+
+    protected void appendToInputString(char input) {
```

Comment:
Perhaps this method should return `this` so it can be chained?

Reply:
It's not something that I've needed.  I'm not opposed to it, but it's a backward-compatible change from `void` to `this` later.  Changing it from `this` to `void` would not be.  I'd prefer to leave that change to someone who needed it.

Reply:
Or perhaps there could instead be a `getAppendableInputString` method that simply returns the field as an `Appendable`, initialising it if needed, and then the caller can use the regular `append` methods.